Add note (#10462)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/set-up-phone-system-voicemail.md

2. In the Run diagnostic pane, enter the email of the user who is having issues receiving voicemail in the **Username or Email** field, and then select **Run Tests**.
3. The tests return the best next steps to address any tenant or policy configurations to validate that the user is properly configured to receive voicemail in Teams.

> >[!NOTE]
> > There could be certain situations where the voicemail form does not get displayed in the UI in Outlook Desktop, the following registry key needs to be set: [HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\Outlook\Addins] "AllowVoicemailForm"=dword:00000001, for users, enterprises, and organizations that want the Voicemail form to appear.
> 

Reviewer: ```suggestion
> [!NOTE]
> There could be certain situations where the voicemail form is not displayed in the UI in Outlook Desktop. The following registry key needs to be set for users, enterprises, and organizations that want the Voicemail form to appear: [HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\Outlook\Addins] "AllowVoicemailForm"=dword:00000001.
```
